lib/sidekiq/instrument/middleware/server.rb

```diff
@@ -16,6 +16,8 @@ def call(worker, job, _queue, &block)
       yield block
       execution_time_ms = (Time.now - start_time) * 1000
       Statter.dogstatsd&.timing('sidekiq.runtime', execution_time_ms, worker_dog_options(worker, job))
+      Statter.statsd.increment(metric_name(worker, 'success'))
+      Statter.dogstatsd&.increment('sidekiq.success', worker_dog_options(worker, job))
       Statter.statsd.measure(metric_name(worker, 'runtime'), execution_time_ms)
```

Comment:
Please keep the order consistent. Same metrics for different tools should go together.
```suggestion
      Statter.dogstatsd&.timing('sidekiq.runtime', execution_time_ms, worker_dog_options(worker, job))
      Statter.statsd.measure(metric_name(worker, 'runtime'), execution_time_ms)
      Statter.dogstatsd&.increment('sidekiq.success', worker_dog_options(worker, job))
      Statter.statsd.increment(metric_name(worker, 'success'))
```

Reply:
Oop sorry hadn't even realized I split them